### ISSUE TITLE
Error: Chromium revision is not downloaded.

### ISSUE THREAD

**Issue by DeividasK** [smartcontractkit/external-adapters-js#89]
### Steps to reproduce
- Switch on Chainlink VPN.
- `curl https://lambda.staging.devnet.tools/googlefinance/call -d '{"data": {"from": "FTSE", "to": "GBP" }}'`.

### Expected outcome
Status code 200 with a response object containing `result` field.

### Actual outcome
`{"jobRunID":"1","status":"errored","error":{"error":{"name":"AdapterError","message":"Chromium revision is not downloaded. Run \"npm install\" or \"yarn install\""}},"statusCode":500}%`

**Comment by krebernisak** [comment]
Looks like we are missing Chromium in a serverless environment. How did this adapter work before?

We could use [alixaxel/chrome-aws-lambda](https://github.com/alixaxel/chrome-aws-lambda) to bundle Chromium Binary for AWS Lambda and Google Cloud Functions. This [How to use Puppeteer in a Netlify (AWS Lambda) function](https://bitsofco.de/how-to-use-puppeteer-in-a-netlify-aws-lambda-function/) looks like a good article on the subject.

@boxhock as you worked on this adapter, what are your thoughts here?

**Comment by DeividasK** [comment]
> How did this adapter work before?
Before (currently in prod) it is running on an AWS ECS instance. We tried to deploy it as a lambda for consistency reasons.

I've also noticed that most adapters are getting a lot of use, which suggests that actual servers would provide better value for money. This would also come with benefits for improved caching and handle use cases where additional requests are received while a cache is empty and at least one request is in flight.

Did we ever consider using docker images deployed on AWS EB?
